internals/schemas/app.schema.json

```diff
@@ -89,6 +89,10 @@
           "description": "When true, will show extra questions after the first page of questions in the incident submission form",
           "type": "boolean"
         },
+        "useDenBoschSignalType": {
```

Comment:
I would prefer not to use client name in the code, how about `useProjectenSingalType`?